scripts/overwrite_representation_tests.sh

```diff
@@ -22,14 +22,13 @@ testFilter="org.jetbrains.kotlinx.lincheck_test.representation.*"
 for jdk in "${jdks[@]}" 
```

Comment:
If jdk 17 is default we should put it first in the `jdks` list. Otherwise we might end up with unnecessary duplicates.